azure-pipelines.yml

```diff
@@ -0,0 +1,105 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+
+workspace:
+    clean: all
+
+variables:
+  - group: "ESFA - SonarCloud"
+  - name: sonarCloudProjectKey
+    value: "SkillsFundingAgency_das-roatp-assessor"
+  - group: 'Prod Management Resources'    
+  - name: buildConfiguration
+    value: 'release'
+  - name: buildPlatform
+    value: 'anycpu'
+
+pool:
+  name: 'DAS - Continuous Integration'
+  demands: 
+  - msbuild
+  - visualstudio
+
+resources:
+  repositories:
+  - repository: das-platform-building-blocks
+    type: github
+    name: SkillsFundingAgency/das-platform-building-blocks
+    ref: refs/heads/master
+    endpoint: "GitHub (SFA)"  
+
+steps:
+- task: gittools.gitversion.gitversion-task.GitVersion@5
```

Comment:
That task has recently been deprecated and we now use UseGitVersion https://github.com/SkillsFundingAgency/das-content-api/blob/master/azure-pipelines.yml#L31

Reply:
Updated